DOCUMENTS/POSTFIX_CONF.txt

```diff
@@ -13,12 +13,19 @@ postconf -m)  Three main.cf variables are involved:
 
 virtual_mailbox_domains = proxy:mysql:/etc/postfix/sql/mysql_virtual_domains_maps.cf
 virtual_alias_maps =
-   proxy:mysql:/etc/postfix/sql/mysql_virtual_alias_maps.cf,
```

Comment:
Hmm, I'm surprised that this fails because IIRC I have more than one "chained" alias ;-) - and I'm even more surprised that changing the order fixed it for you.

The setting that will probably break is:
* alias a@host -> foo@example.com
* alias a@otherhost -> bar@example.com
* alias b@otherhost -> b@example.com (and some other aliases that don't exist @host)
* alias domain host -> otherhost

The goal is: all mails that are sent to @host should be forwarded to the same address @otherhost (if this address exists) - except a@ because the a@host alias overrides this.